core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutorModule.java

```diff
@@ -21,7 +21,7 @@
 public class WorkflowExecutorModule extends AbstractModule {
     @Override
     protected void configure() {
-        bind(WorkflowStatusListener.class).to(WorkflowStatusListenerStub.class);//default implementation
+        //bind(WorkflowStatusListener.class).to(WorkflowStatusListenerStub.class);//default implementation
```

Comment:
Is this conflicting with our listener?

Reply:
Yes this also implements the same listener